x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/mapper/ShapeFieldMapperTests.java

```diff
@@ -72,6 +72,12 @@ protected void registerParameters(ParameterChecker checker) throws IOException {
         });
     }
 
+    @Override
```

Comment:
I am missing here explicit test for doc_values, like that they are generated when set the flag to true or that the default value is true in current version but. false in previous versions.

Reply:
Added in 1066e01c98b3d8f5bd9873803366d5a2d7785f80